delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/QuotasRoutes.scala

```diff
@@ -41,7 +38,7 @@ final class QuotasRoutes(
   def routes: Route =
     (baseUriPrefix(baseUri.prefix) & pathPrefix("quotas")) {
       extractCaller { implicit caller =>
-        projectRef(projects).apply { implicit ref =>
+        projectRef.apply { ref =>
```

Comment:
No need to fetch quotas via project uuid (the idea is also to align quotas on how we deal with acls)